File: testresources/csharp/benchmark/Domain/Direct/Violating/AccessInstanceVariableSuperSuperClass.cs
using Technology.Direct.Subclass;

namespace Domain.Direct.Violating
{
	public class AccessInstanceVariableSuperSuperClass
	{
		private CallInstanceSubSubClassDOA subSubDao;

		public String Method()
		{
			return subSubDao.VariableOnSuperClass;
		}
	}
}

File: testresources/java/accuracy/src/domain/direct/allowed/AccessInstanceVariableSuperSuperClass.java
package domain.direct.allowed;

import domain.direct.Base;

public class AccessInstanceVariableSuperSuperClass extends Base{
	
	public AccessInstanceVariableSuperSuperClass(){};
	
	public void Method() {
		String s = subSubDao.VariableOnSuperClass; 
		System.out.println(s);
	}
}
